lldb/source/Interpreter/CommandObject.cpp

```diff
@@ -769,10 +768,10 @@ bool CommandObjectRaw::Execute(const char *args_string,
     handled = InvokeOverrideCallback(argv, result);
   }
   if (!handled) {
-    if (CheckRequirements(result))
-      handled = DoExecute(args_string, result);
+    if (CheckRequirements(result)) {
+      DoExecute(args_string, result);
+    }
```

Comment:
No braces around single line if's as per the [LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements).

Reply:
I took out the braces.
Interesting the code_formatter test/bot didn't flag that or the other one-liners within curly braces.

I typically write my code for safety first and then readability/maintainability, and I put in the curly braces in for those reasons.